Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -649,9 +649,6 @@ const ScrollView = React.createClass({
         {...contentSizeChangeProps}
         ref={this._setInnerViewRef}
         style={contentContainerStyle}
-        removeClippedSubviews={
-          hasStickyHeaders && Platform.OS === 'android' ? false : this.props.removeClippedSubviews
-        }
```

Comment:
You're right, I should have actually checked the diff -_- https://github.com/facebook/react-native/pull/11315/files#diff-f8f8b220fc0d1573e4f8b78c84415075L478